frontend/src/components/browse-sightings-page/MapPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sighting } from "../../clients/apiClient";
+// import { SlippyMap, Marker, Label, InfoBox } from "react-slippy-map";
+import GoogleMapReact from "google-map-react";
+import { Marker } from "./marker";
+import React from "react";
+import { map } from "leaflet";
+import "./mapPage.scss";
+
+interface MapPageProps {
+  sightingList: Sighting[];
```

Comment:
The word "list" is usually more concisely expressed in variable names by using the plural form of the noun
```suggestion
  sightings: Sighting[];
```